src/hooks/useKeyboardNavigation.js

```diff
@@ -54,8 +54,7 @@ const useKeyboardNavigation = ({
   }, []);
 
   const updateActiveItem = newActiveItem => {
-    if (newActiveItem && newActiveItem.firstChild)
-      focusElement(newActiveItem.firstChild);
+    if (newActiveItem?.firstChild) focusElement(newActiveItem.firstChild);
```

Comment:
Would it make sense to forward this condition down to `focusElement`?
Then focusElement will do all the viability checks. I see we're repeating the same condition in every place we're calling focusElement...
```suggestion
    focusElement(newActiveItem?.firstChild);
```